packages/react-components/lib/dispensers/dispenser-accordion.tsx

```diff
@@ -48,18 +49,21 @@ const useStyles = makeStyles((theme) => ({
   statusLabelOffline: { borderColor: theme.palette.error.main },
 }));
 
+// for valid dispenser states,
+// a precondition must dispenser === dispenserState.guid
 export interface DispenserAccordionProps extends Omit<AccordionProps, 'children'> {
-  dispenserState: RomiCore.DispenserState;
+  dispenserState: RomiCore.DispenserState | null;
```

Comment:
```suggestion
  /**
   * Pre-condition: `dispenser === dispenserState.guid`
   */
   dispenserState: RomiCore.DispenserState | null;
```

Reply:
Didn't know that this can be done!

Reply:
Is there a link to all the possible syntax? I can't seem to find the correct docs on vscode

Reply:
vscode tries to find the docs from several sources, one of which is jsdoc.